diff --git a/node_modules/vue3-simple-typeahead/src/vue3-simple-typeahead.vue b/node_modules/vue3-simple-typeahead/src/vue3-simple-typeahead.vue
index d96c7af..5edeed2 100644
--- a/node_modules/vue3-simple-typeahead/src/vue3-simple-typeahead.vue
+++ b/node_modules/vue3-simple-typeahead/src/vue3-simple-typeahead.vue
@@ -1,8 +1,9 @@
 <template>
 	<div :id="wrapperId" class="simple-typeahead">
+	<h1>Hello</h1>
 		<input
 			:id="inputId"
-			class="simple-typeahead-input"
+			class="uk-input"
 			type="text"
 			:placeholder="placeholder"
 			v-model="input"
@@ -36,8 +37,9 @@
 </template>
 
 <script>
+	import "./../node_modules/uikit/dist/js/uikit.min.js";
+	import "./../node_modules/uikit/dist/js/uikit-icons.min.js"
 	import { defineComponent } from 'vue';
-
 	export default /*#__PURE__*/ defineComponent({
 		name: 'Vue3SimpleTypeahead',
 		emits: ['onInput', 'onFocus', 'onBlur', 'selectItem'],
@@ -166,6 +168,7 @@
 </script>
 
 <style scoped>
+	@import "./../node_modules/uikit/dist/css/uikit.min.css";
 	.simple-typeahead {
 		position: relative;
 		width: 100%;
